channels/ve.m3u

```diff
@@ -1,51 +1,61 @@
 #EXTM3U
-#EXTINF:-1 tvg-id="123TV.ve" tvg-name="123 TV" tvg-country="VE" tvg-language="Spanish" tvg-logo="" group-title="",123 TV
```

Comment:
Maybe it would be better to add `[Offline]` to leave a description in the playlist?

Reply:
I'm quite unsure as if it'll come back online, as it's a Astra CP link.